app/nexus.go

```diff
@@ -97,11 +99,15 @@ func (c *NexusClient) createEndpoint(
 
 func (c *NexusClient) patchEndpoint(
 	existingEndpoint *nexus.Endpoint,
+	description string,
 	targetNamespaceID string,
 	targetTaskQueue string,
 	resourceVersion string,
 	asyncOperationId string,
 ) (*operation.AsyncOperation, error) {
+	if description != "" {
```

Comment:
what's the tcld command to clear an existing description?

Reply:
Added ` nexus endpoint description set/unset` commands.
Now we have 2 ways of updating the description.
- Wanted to keep it in the `update` command to be consistent with `create`
- Wanted to keep the `set` to be consistent with `unset`

Happy to change if you think one of the ways is unnecessary.

Reply:
Changed this too `unset-description` bool flag similar to os cli.
https://github.com/bergundy/cli/blob/187b82da3e3d8702df476bc418abf9b706f7f95a/temporalcli/commands.gen.go#L979C5-L979C56